test/test-pages/article-author-tag/expected-metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "If You Can Picture A Tarot Card, It's Because of These 3 People",
+  "byline": "Laura June Topolsky",
```

Comment:
This is the key line. Without this change, this byline would contain something like "Laura June Topolsky July 10, 2015".